### ISSUE TITLE
Extend availbility metric

### ISSUE THREAD

**Issue by tomerweller** [stellarbeat/js-stellarbeat-frontend#4]
Currently it's based on 300 crawls, considering crawls are ~4 minutes this translates to ~20 hours.

Less than a day is not a good enough metric for the historic health of a node. Also, number of crawls is not a deterministic time frame which makes that a bit hard to parse.

I suggest using 30 days as the timeframe for availability.

**Comment by pieterjan84** [comment]
Proposal: 30 day & 24 hour percentages for the following metrics: 
- online: the node is online, we can perform a successful handshake.
- validating: the node is participating in scp (externalizing messages).
- overloaded: the node disconnects due to high load

I would drop 'availability' and add the metrics above in the node index calculation.

**Comment by pieterjan84** [comment]
24H and 7D stats available. Will switch to 30D when enough data is available.

**Comment by pieterjan84** [comment]
Switched 7D to 30D statistics.
